api/spec/requests/spree/api/v2/storefront/stores_controller_spec.rb

```diff
@@ -0,0 +1,40 @@
+require 'spec_helper'
+
+describe 'Storefront API v2 Stores spec', type: :request do
+  let!(:store) { create(:store) }
+
+  describe 'stores#show' do
+    context 'with code param' do
+      before { get "/api/v2/storefront/stores/#{store.code}" }
+
+      it 'return store with attributes' do
+        expect(json_response['data']).to have_attribute(:name).with_value(store.name)
+        expect(json_response['data']).to have_attribute(:url).with_value(store.url)
+        expect(json_response['data']).to have_attribute(:meta_description).with_value(store.meta_description)
+        expect(json_response['data']).to have_attribute(:meta_keywords).with_value(store.meta_keywords)
+        expect(json_response['data']).to have_attribute(:seo_title).with_value(store.seo_title)
+        expect(json_response['data']).to have_attribute(:default_currency).with_value(store.default_currency)
+        expect(json_response['data']).to have_attribute(:default).with_value(store.default)
+        expect(json_response['data']).to have_attribute(:supported_currencies).with_value(store.supported_currencies)
+        expect(json_response['data']).to have_attribute(:facebook).with_value(store.facebook)
+        expect(json_response['data']).to have_attribute(:twitter).with_value(store.twitter)
+        expect(json_response['data']).to have_attribute(:instagram).with_value(store.instagram)
+        expect(json_response['data']).to have_attribute(:default_locale).with_value(store.default_locale)
+        expect(json_response['data']).to have_attribute(:customer_support_email).with_value(store.customer_support_email)
+        expect(json_response['data']).to have_attribute(:default_country_id).with_value(store.default_country_id)
```

Comment:
@przemosk we should not return IDs per se, this should be handled by `has_one :default_country`

Reply:
that mean you want to change thist attribute from `default_country_id` to `default_country` and return Contry Object instead of just `id`  ?